src/structures/MessageAttachment.js

```diff
@@ -103,6 +103,12 @@ class MessageAttachment {
      * @type {?string}
      */
     this.contentType = data.content_type ?? null;
+
+    /**
+     * Whether this attachment is ephemeral
+     * @type {?boolean}
+     */
+    this.ephemeral = data.ephemeral ?? null;
```

Comment:
I believe `data.ephemeral` is only sent when true, so shouldn't this default to false instead?

Reply:
yeah, I guess so. What do others think should be done here? 👀

Reply:
Personally I agree. @discordjs/the-big-3?

Reply:
Yes.